7_deploy_osh/roles/suse-osh-setup/tasks/apply-certs.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Please remove this part of the change too (https://github.com/SUSE-Cloud/socok8s/pull/93)